cli/command/system/version.go

```diff
@@ -189,3 +211,32 @@ func getDetailsOrder(v types.ComponentVersion) []string {
 	sort.Strings(out)
 	return out
 }
+
+func getKubernetesVersion(dockerCli command.Cli, cmd *cobra.Command) (*kubernetesVersion, error) {
+	if !dockerCli.ClientInfo().HasKubernetes() {
+		return nil, nil
+	}
+
+	kli, err := kubernetes.WrapCli(dockerCli, cmd)
```

Comment:
Instead of `WrapCLI()` if this could call a method that returns `KubeConfig` then it can be used for both `GetAPIVersion` and getting a `kubeClient` without the need for `KubeCli`.

Reply:
Done